README.md

```diff
@@ -3,13 +3,16 @@
 [![Travis CI status][travis-shield]][travis-link]
 [![AppVeyor CI status][appveyor-shield]][appveyor-link]
 [![Codecov Coverage status][codecov-shield]][codecov-link]
+[![Documentation][doxygen-shield]][doxygen-link]
```

Comment:
This is fine, but one issue we have is that our current docs treat "Google Cloud C++" as being synonymous with "Bigtable". This is the case today, but "google-cloud-cpp" is the name of the module, so Bigtable should be one of the modules, whereas it appears that it would be very hard to retrofit another product here.

It should be titled as "Google Cloud C++ libraries" and "Bigtable" should be the only product currently listed, but we should have more products here later in 2018, so we should enable it soon, so that our users don't get dependent on a particular page layout or URLs mapping directly to Bigtable.

Thoughts?

Reply:
I agree.  Created #243 to track the change of layout in the documentation.